docs/source/datamodules_others.rst

```diff
@@ -0,0 +1,69 @@
+.. role:: hidden
```

Comment:
Why do we need this? We already have https://lightning-bolts.readthedocs.io/en/latest/datamodules_vision.html.

Reply:
@akihironitta Thank you, for pointing it out. When I looked at the documentation, I could not find the modules (only Vision and Sklearn modules were there) on the navigation pane. Is there a way to make the modules appear/collapse on the navigation pane?

I will add `EMNISTDataModule` and `BinaryEMNISTDataModule` to `datamodules_vision.rst` and remove `datamodules_others.rst`.

Reply:
:bulb: The latest update is posted above [here](https://github.com/PyTorchLightning/lightning-bolts/pull/676#issuecomment-875279745).